.github/workflows/perform-release.yml

```diff
@@ -0,0 +1,83 @@
+# Performing a release is done by executing a perform-release event through a repository webhook.
+# This needs to be executed by issuing an authenticated REST Post operation, to prevent accidental or unauthorized releases.
+# This should involve a command such as the following:
+#
+#    export GITHUB_TOKEN=your-personal-access-token
+#    curl  -H "Authorization: token $GITHUB_TOKEN" \
+#      -H "Accept: application/vnd.github.everest-preview+json"  \
+#      -H "Content-Type: application/json" \
+#      --request POST \
+#      --data '{ "event_type": "perform-release", "client_payload": { "ref": "master" } }' \
+#      https://api.github.com/repos/Rwanda-EMR/openmrs-distro-rwandaemr/dispatches
```

Comment:
This comment line needs to be adjusted to https://api.github.com/repos/Rwanda-EMR/openmrs-module-pharmacymanagement/dispatches

Reply:
Done.

Reply:
Thanks @mseaton . This is merged.